minitap/mobile_use/context.py

```diff
@@ -19,6 +19,14 @@
 from minitap.mobile_use.config import AgentNode, LLMConfig
 
 
+class AppLaunchResult(BaseModel):
+    """Result of initial app launch attempt."""
+
+    locked_app_package: str | None
```

Comment:
If you send a result, then we are 100% sure that we have an app package to lock (cf my previous comment)
So, why can't we simplify this, locked_app_package should not be optional right

```suggestion
    locked_app_package: str
```